src/main/resources/templates/addBloodDrive.html

```diff
@@ -0,0 +1,205 @@
+
+			  <div class="ui placeholder inverted segment" id="fogCover" style="background: rgba(0,0,0,.7); max-width: 1200px; max-height: 475px; overflow-y: auto;">
+                <h1 style="font-size:4vw;">Add Blood Drive</h1>
+                <p style="font-size:2vw;">Make sure the information below is correct</p>
+
+                <form class="ui inverted form">
+                  <div class="field">
+                    <label>Title:</label>
+                    <input type="text" id="title" placeholder="title">
+                  </div>
+                  <div class="field">
+                    <label>Location:</label>
+                    <input type="text" id="location" placeholder="Location">
+                  </div>
+                  <div class="field">
+                    <label>Time:</label>
+                    <input type="time" id="time">
+                  </div>
+                  <div class="field">
+                    <label>Date:</label>
+                    <input type="date" id="date">
+                  </div>
+                  <div class="field">
+                    <label>Description:</label>
+                    <textarea id="description" placeholder="Description"></textarea>
+                  </div>
+                  <div class="field">
+                    <label>Link:</label>
+                    <input type="text" id="link" placeholder="link">
+                  </div>
+                </form>
+                <br>
+                <button id="addBloodDriveButton" class="ui button primary">Add Blood Drive</button>
+
+              </div>
+            </div>
+        </div>
+
+    </div>
+ </div>
+</div>
+<script>
+  $(document).ready(function() {
+    $.get("/loginStatus", function(data, status) {
+      if (data.isLoggedIn == true){
+        document.getElementById("logoutButton").style.display = "block";
+        document.getElementById("loginButton").style.display = "none";
+      }
+
+
+    });
+  });
+
+
+  function inputCheck() {
+    var isError = false;
+    var titleLen = (($("#title").val()).length)
+    var locationLen = (($("#location").val()).length);
+    var timeLen = (($("#time").val()).length) + 3;
+    var dateLen = (($("#date").val()).length);
+    var descriptionLen = (($("#description").val()).length);
+
+    if (titleLen == 0) {
+      document.getElementById("title").style.backgroundColor = "#FA8072";
+      isError = true;
+    }
+    if (locationLen == 0) {
+      document.getElementById("location").style.backgroundColor = "#FA8072";
+      isError = true;
+    }
+    if (timeLen != 8) {
+      document.getElementById("time").style.backgroundColor = "#FA8072";
+      isError = true;
+    }
+    if (dateLen != 10) {
+      document.getElementById("date").style.backgroundColor = "#FA8072";
+      isError = true;
+    }
+    if (descriptionLen == 0) {
+      document.getElementById("description").style.backgroundColor = "#FA8072";
```

Comment:
this might be better in a method call were you send the id as input. Since it's doing the same thing for all of those fields.